ehr/api-src/org/labkey/api/ehr/SharedEHRUpgradeCode.java

```diff
@@ -216,7 +224,22 @@ public void moduleStartupComplete(ServletContext servletContext)
             {
                 for (TsvImport tsvImport : _tsvImports)
                 {
-                    importFile(tsvImport, container, user);
+                    if (tsvImport._containerPath != null)
+                    {
+                        Container tsvImportContainer = ContainerManager.getForPath(tsvImport._containerPath);
+                        if (tsvImportContainer != null)
+                        {
+                            importFile(tsvImport, ContainerManager.getForPath(tsvImport._containerPath), user);
+                        }
+                        else
+                        {
+                            importFile(tsvImport, container, user);
```

Comment:
Let's log that we couldn't find the target container and are falling back to the default.

Reply:
done